go.mod

```diff
@@ -25,8 +25,8 @@ require (
 	github.com/jinzhu/gorm v1.9.12
 	github.com/kelseyhightower/envconfig v1.4.0 // indirect
 	github.com/lib/pq v1.3.0
-	github.com/lyft/flyteidl v0.18.6
-	github.com/lyft/flytepropeller v0.3.17
+	github.com/lyft/flyteidl v0.18.7
+	github.com/lyft/flytepropeller v0.4.2-0.20201001203502-c9d4b2a9b8dd
```

Comment:
let's wait until the propeller changes are in (with an official release or tag) before merging this

Reply:
sure